Demo/SceneController.swift

```diff
@@ -7,50 +7,24 @@ final class SceneController: UIResponder {
     var window: UIWindow?
 
     private let rootURL = Demo.current
-    private lazy var navigator = Navigator(pathConfiguration: pathConfiguration, delegate: self)
-
-    // MARK: - Setup
-
-    private func configureBridge() {
-        Hotwire.registerBridgeComponents([
-            FormComponent.self,
-            MenuComponent.self,
-            OverflowMenuComponent.self,
-        ])
-    }
-
-    private func configureRootViewController() {
-        guard let window = window else {
-            fatalError()
-        }
-
-        window.rootViewController = navigator.rootViewController
-    }
+    private lazy var navigator = Navigator(delegate: self)
```

Comment:
This no longer needs to be `lazy`, correct?

Reply:
It still needs to be `lazy` because we are referencing `self` before `self` is instantiated.